reporters/reporter-default/src/main/java/org/radargun/reporting/html/ReportDocument.java

```diff
@@ -59,114 +62,28 @@ public ReportDocument(String targetDir, String testName, int maxConfigurations,
-      final String chartTableHeaderBegin = "<th style=\"text-align: center;border: 0px;\">";
-      final String chartTableHeaderEnd = "</th>\n";
-
-      String suffix = clusterSize > 0 ? "_" + clusterSize : "";
-      StringBuilder headerString = new StringBuilder("<table>\n");
-      StringBuilder chartString = new StringBuilder();
-      String directory = this.directory.endsWith(File.separator) ? this.directory : this.directory + File.separator;
-
-      String responseTimeChartFile = String.format("%s_%s%s_mean_dev.png", testName, operation, suffix);
-      boolean hasResponseTimeChart = createChart(directory + responseTimeChartFile,
-            clusterSize, operation, "Response time (ms)", ChartType.MEAN_AND_DEV);
-      if (hasResponseTimeChart) {
-         headerString.append(chartTableHeaderBegin).append("Response time mean").append(chartTableHeaderEnd);
-         chartString.append(String.format(chartTableCellFormat, responseTimeChartFile, operation));
-      }
-      String throughputGrossChartFile = String.format("%s_%s%s_throughput_gross.png", testName, operation, suffix);
-      boolean hasOperationThroughputGrossChart = createChart(directory + throughputGrossChartFile,
-            clusterSize, operation, "Operations/sec", ChartType.OPERATION_THROUGHPUT_GROSS);
-      if (hasOperationThroughputGrossChart) {
-         headerString.append(chartTableHeaderBegin).append("Gross operation throughput").append(chartTableHeaderEnd);
-         chartString.append(String.format(chartTableCellFormat, throughputGrossChartFile, operation));
-      }
-      String throughputNetChartFile = String.format("%s_%s%s_throughput_net.png", testName, operation, suffix);
-      boolean hasOperationThroughputNetChart = createChart(directory + throughputNetChartFile,
-            clusterSize, operation, "Operations/sec", ChartType.OPERATION_THROUGHPUT_NET);
-      if (hasOperationThroughputNetChart) {
-         headerString.append(chartTableHeaderBegin).append("Net operation throughput").append(chartTableHeaderEnd);
-         chartString.append(String.format(chartTableCellFormat, throughputNetChartFile, operation));
-      }
-      final String dataThroughputChartFile = String.format("%s_%s%s_data_throughput.png", testName, operation, suffix);
-      boolean hasDataThroughputChart = createChart(directory + dataThroughputChartFile,
-            clusterSize, operation, "MB/sec", ChartType.DATA_THROUGHPUT);
-      if (hasDataThroughputChart) {
-         headerString.append(chartTableHeaderBegin).append("Data throughput mean").append(chartTableHeaderEnd);
-         chartString.append(String.format(chartTableCellFormat, dataThroughputChartFile, operation));
-      }
-
-      headerString.append("<tr>\n");
-      write(headerString.toString());
-      write(chartString.toString());
-      write("</tr></table>\n");
+   public int getMaxClusters() {
+      return maxClusters;
```

Comment:
That's maximum number of clusters taken from test configuration[2,4,6,8,...] or what?